src/handleActions.js

```diff
@@ -1,10 +1,11 @@
 import handleAction from './handleAction';
 import ownKeys from './ownKeys';
-import reduceReducers from 'reduce-reducers';
 
 export default function handleActions(handlers, defaultState) {
   const reducers = ownKeys(handlers).map(type => handleAction(type, handlers[type]));
-  const reducer = reduceReducers(...reducers);
+  const reducer = (state, action, ...args) => reducers.reduce(
+    (p, r) => r(p, action, ...args), state);
```

Comment:
I don't like these one-letter variable names.

Can we please use: 

```js
reducers.reduce((previousState, reducer) => reducer(previousState, action, ...args), state)
```

Also, at this point, we should remove `reducer-reducers` from package.json, since we're no longer using it. 

And, could you submit a pull request [here](https://github.com/acdlite/reduce-reducers) for passing the extra arguments?